model_analyzer/plots/plot_manager.py

```diff
@@ -83,7 +84,7 @@ def _create_summary_plot_for_model(self, model_name, plots_key,
         """
 
         for plot_config in self._config.plots:
-            constraints = self._constraints['default']
+            constraints = self._constraints['__default__']
```

Comment:
Modified key to access global constrains as `__default__` to look more unique than model names, as model name will not start with `_`

Reply:
We should make this a define. Can you add this to `constants.py`?

Reply:
Defined `GLOBAL_CONSTRAINTS_KEY = "__default__"` in `constants.py`